### ISSUE TITLE
Feature request : list Areas that are easily accessible by public transport 🚲 🚃

### ISSUE THREAD

**Issue by vviers** [boolder-org/boolder-data#3]
Salut Nicolas et l'équipe 👋🏼 

As a climber who's trying to limit the environmental impact of my climbing, I often end up going to Bleau using a mix of train + walking/biking to the crag. I think listing those areas explicitly (along with route information from Paris for example) would be a great addition to the Area database.

I'm thinking a `no_car_needed` category in the tag attribute along with a new `how_to_get_there` would fit the bill.

Wdyt ? Happy to contribute if you're interested in this :)

**Comment by nmondollot** [comment]
Hello Vincent!

Thanks for the suggestion!
Train+bike is awesome 👍 

I already created a dedicated page on the website ([here](https://www.boolder.com/fr/articles/top-areas/train)) and on the iPhone app. It will come later on the Android app.
Do you think we should include it in the `boolder-data` database too? Or do you just want to improve the UI on the website and/or the apps? Ideas are welcome :)

FYI I'm also working on a way to make the train stations more visible directly on the map.

**Comment by vviers** [comment]
> Do you think we should include it in the `boolder-data` database too

Yes, I think that would be very valuable information for initiatives wishing to re-use this data :) they are also meaningful attributes of a sector imo

> I already created a dedicated page on the website ([here](https://www.boolder.com/fr/articles/top-areas/train))

This is so awesome 👌🏼 thanks again for the mega work, and I'm more than happy to help if I can !

**Comment by nmondollot** [comment]
> Yes, I think that would be very valuable information for initiatives wishing to re-use this data :) they are also meaningful attributes of a sector imo

All right, I will do it then :)

> This is so awesome 👌🏼 thanks again for the mega work, and I'm more than happy to help if I can !

Thanks 🙏 ❤️ 
Let's chat! Drop me a line at hello@boolder.com
